modules/docs/arrow-docs/docs/docs/arrow/data/mapk/README.md

```diff
@@ -11,13 +11,82 @@ redirect_from:
 {:.beginner}
 beginner
 
-### Supported type classes
+MapK is an Arrow wrapper over Kotlin Map type. The main goal is to make it a [type constructor](/docs/patterns/glossary/#type-constructors)
```

Comment:
Also check the format to link other sub-documents, IIRC it's more like `{{ | }}`